src/style.css

```diff
@@ -2,0 +1,86 @@
+@font-face {
+  font-family: Avenir, Arial, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy, Arial, sans-serif;
```

Comment:
The same issue as above. The `font-family` property inside a `@font-face` rule should contain only the name of the font that's being loaded with the rule.